crates/storage/store/storage.rs

```diff
@@ -32,6 +35,7 @@ pub struct Store {
     engine: Arc<dyn StoreEngine>,
     pub mempool: Arc<Mutex<HashMap<H256, MempoolTransaction>>>,
     pub blobs_bundle_pool: Arc<Mutex<HashMap<H256, BlobsBundle>>>,
+    tx_pending_requests: Arc<Mutex<PendingRequests>>,
```

Comment:
I don't think this is the place to store this. It is specific to the p2p layer and I don't think anyone else needs access to this info. I'd prefer if it was encapsulated inside the p2p crate.